src/docs/plain.ml

```diff
@@ -77,6 +77,16 @@ let plain_of_func_sort : Buffer.t -> Syntax.func_sort -> unit =
     | Shared Query -> bprintf buf "shared query "
     | Shared Write -> bprintf buf "shared ")
 
+let plain_of_obj_sort_title : Buffer.t -> Syntax.obj_sort -> unit =
+ fun buf sort ->
+  Buffer.add_string buf
+    Mo_types.Type.(
+      match sort.it with
+      | Object -> "Object "
+      | Actor -> "Actor "
+      | Module -> "Module "
+      | Memory -> "Memory ")
```

Comment:
```suggestion
      | Memory -> "Memory ")
```
This case shouldn't really arise but fine to support it.

Reply:
Makes sense. For some reason this was already included, so I just kept the original logic.